Directory.Build.props

```diff
@@ -42,13 +42,18 @@
   <!-- Shared Package Versions -->
   <PropertyGroup>
     <!-- System packages -->
+    <SystemCollectionsImmutableVersion>5.0.0</SystemCollectionsImmutableVersion>
```

Comment:
Could move these to a separate block with a comment about these being for only netstandard2.0 (like it was before my last major cleanup that removed these).